docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -207,6 +207,22 @@ This means that the number of rows will drive the height of the grid and consequ
 
 {{"demo": "pages/components/data-grid/rendering/AutoHeightGrid.js", "bg": "inline"}}
 
+## Density
+
+You can change the density of the rows and the column header.
+
+### Density prop
+
+The vertical density of the data grid can be set using the `density` prop. Regardless of its setting, the `density` prop will respect values provided using the `rowHeight` or `headerHeight` props.
+
+{{"demo": "pages/components/data-grid/rendering/DensitySelectorSmallGrid.js", "bg": "inline"}}
+
+### Density selector button
+
+The default way for a user to change the density of the data grid is by using the density selector component from the toolbar. In addition, there is a prop to hide the density selector.
```

Comment:
```suggestion
The user can change the density of the data grid  by using the density selector from the toolbar. This can be hidden with the `disableDensitySelector` prop.
```

Reply:
Should this be called `hideDensitySelector` to better reflect it's function?

![image](https://user-images.githubusercontent.com/357702/101086127-4f87a680-35a8-11eb-96e2-4b478e7b8e92.png)

Reply:
IMHO, it would be better to drop these props and move to the opt-in toolbar + composable API, one of the last few points of #633 that isn't checked yet.

Reply:
We can remove them later. For now the name matches the name of the other `toolbar` props.